patches/net-base-host_port_pair.cc.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/net/base/host_port_pair.cc b/net/base/host_port_pair.cc
+index ec8248449dbcd4156e361fcc8781ce0712f26a02..77f5a3653083625f8043147d42297dba05bb79a9 100644
+--- a/net/base/host_port_pair.cc
++++ b/net/base/host_port_pair.cc
+@@ -18,9 +18,11 @@
+ namespace net {
+ 
+ HostPortPair::HostPortPair() : port_(0) {}
++#if 0                           // brave override
```

Comment:
these should use BRAVE_CHROMIUM_BUILD

Reply:
5e8611a27cd370eb31c4b0e911a38e9fc0341c9f